recipes/roaring/all/conanfile.py

```diff
@@ -10,15 +12,24 @@ class ConanRecipe(ConanFile):
 
     homepage = "https://github.com/RoaringBitmap/CRoaring"
     url = "https://github.com/conan-io/conan-center-index"
-
     license = "Apache-2.0"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "avx": [True, False],
+        "neon": [True, False],
```

Comment:
This should not be needed.

Reply:
The library does not actually make use of explicit neon instructions at all to my recollection, so it should be straight C/C++ under any ARM target and there should be no need to try to detect NEON.